Jenkinsfile

```diff
@@ -167,34 +167,40 @@ node('ubuntu-zion-legacy') {
         OsTools.runSafe(this, "cp -n '${env.HOME}/.docker/config.json' '${env.WORKSPACE_TMP}/.dockerConfig' || true")
         withEnv(["DOCKER_CONFIG=${env.WORKSPACE_TMP}/.dockerConfig", 'DOCKER_CONTENT_TRUST=1']) {
           withCredentials([
-              string(credentialsId: 'nexus-iq-server_dct_reg_pw', variable: 'DOCKER_CONTENT_TRUST_REPOSITORY_PASSPHRASE'),
-              string(credentialsId: 'sonatype_docker_root_pw', variable: 'DOCKER_CONTENT_TRUST_ROOT_PASSPHRASE'),
-              file(credentialsId: 'nexus-iq-server_dct_gun_key', variable: 'DELEGATION_KEY'),
-              file(credentialsId: 'sonatype_docker_root_public_key', variable: 'PUBLIC_KEY'),
+              file(credentialsId: 'nexus-iq-server-repository-key', variable: 'NEXUS_IQ_SERVER_REPOSITORY_KEY'),
+              file(credentialsId: 'sonatype-pub', variable: 'SONATYPE_PUB'),
+              file(credentialsId: 'sonatype-key', variable: 'SONATYPE_KEY'),
               [$class: 'UsernamePasswordMultiBinding', credentialsId: 'docker-hub-credentials',
                usernameVariable: 'DOCKERHUB_API_USERNAME', passwordVariable: 'DOCKERHUB_API_PASSWORD']
           ]) {
-
-            OsTools.runSafe(this, "docker tag ${imageId} ${organization}/${dockerHubRepository}:${version}")
-            OsTools.runSafe(this, "docker tag ${imageId} ${organization}/${dockerHubRepository}:latest")
-            OsTools.runSafe(this, "docker tag ${slimImageId} ${organization}/${dockerHubRepository}:${version}-slim")
-            OsTools.runSafe(this, "docker tag ${slimImageId} ${organization}/${dockerHubRepository}:latest-slim")
-
             OsTools.runSafe(this, """
             docker login --username ${env.DOCKERHUB_API_USERNAME} --password ${env.DOCKERHUB_API_PASSWORD}
             """)
 
-            // Add delegation private key
-            OsTools.runSafe(this, 'docker trust key load $DELEGATION_KEY --name sonatype')
+            // load the repository key..
+            OsTools.runSafe(this, 'docker trust key load $NEXUS_IQ_SERVER_REPOSITORY_KEY')
 
-            // Add delegation public key
-            OsTools.runSafe(this, "docker trust signer add --key $PUBLIC_KEY sonatype ${organization}/${dockerHubRepository}")
+            // load the signers private key
+            OsTools.runSafe(this, 'docker trust key load $SONATYPE_KEY')
+
+            // add signer - for this you need signers public key and repository keys password
+            withCredentials([string(credentialsId: 'nexus-iq-server_dct_reg_pw', variable: 'DOCKER_CONTENT_TRUST_REPOSITORY_PASSPHRASE')]) {
```

Comment:
in another iteration after the next successful release, it might be good to rename this credential like you did others. you have the experience to give it a better name.

Reply:
Once we prove it works, we should also remove any credentials under https://jenkins.ci.sonatype.dev/job/insight/job/insight-brain/job/docker/credentials/store/folder/domain/_/ that are no longer needed. I assume some were created during testing.